FinniversKit/Sources/SwiftUI Components/Helpers/SwiftUIHostingTableViewCell.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+public class SwiftUIHostingTableViewCell<Content: View>: UITableViewCell {
+
+    private var controller: UIHostingController<Content>?
+
+    public func host(_ view: Content, parent: UIViewController? = nil) {
+
+        if let controller = controller {
+            controller.rootView = view
+            controller.view.layoutIfNeeded()
+        } else {
+            let controller = UIHostingController(rootView: view)
+            controller.view.translatesAutoresizingMaskIntoConstraints = false
+            controller.view.backgroundColor = .clear
+            self.controller = controller
+            
+            parent?.addChild(controller)
+            
+            addSubview(controller.view)
```

Comment:
Is it correct that both of these are added directly to the cell and not their `contentView`?